.github/workflows/release.yml

```diff
@@ -186,8 +186,8 @@ jobs:
         if: steps.targets.outputs.hub == 'unstable' && steps.targets.outputs.upload == 'true'
         run: |
           tag=${GITHUB_REF##*/}
-          docker tag tykio/tyk-gateway tykio/tyk-gateway:${{ steps.targets.outputs.hub }}
-          docker tag tykio/tyk-gateway tykio/tyk-gateway:${tag}
+          docker tag tykio/tyk-gateway:${tag} tykio/tyk-gateway:${{ steps.targets.outputs.hub }}
+          docker tag tykio/tyk-gateway:${tag} tykio/tyk-gateway:${tag}
```

Comment:
This looks odd... what's the purpose?

Reply:
Since goreleaser does not build the latest tag anymore, all the commands that used to operate on the default latest need an explicit tag.

Reply:
Removed the unnecessary tagging step.